parustiko/src/protocol/binary_packet.rs

```diff
@@ -119,6 +123,18 @@ impl DecodeRaw for BinaryProtocolPacket {
     }
 }
 
+impl Decode for BinaryProtocolPacket {
+    type Entity = Self;
+
+    fn from_be_bytes(buffer: Vec<u8>) -> Result<Self::Entity, BppError> {
+        let mut cursor = std::io::Cursor::new(buffer);
+        println!("{:?}", cursor);
```

Comment:
```suggestion
```

Reply:
btw we could add checker script for prints in CI